woocommerce/woocommerce-ios
# [Mobile Payments] Order storage update to add ChargeID

Part of: #5976


### Description


Adds the `chargeID` attribute to the Core Data model and includes the related model changes in Storage. `chargeID` was added to the Networking model in #6008, this is split out for clarity.

### Testing instructions

Run unit tests.
Launch the app and use Orders to check that the data migration did not fail.

### Screenshots

N/A no user facing changes

---
